olive/passes/onnx/quantization.py

```diff
@@ -611,23 +618,65 @@ def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassCon
             "algorithm": PassConfigParam(
                 type_=str,
                 default_value=None,
-                description="Available from onnxruntime>=1.17.0 "
-                "Algorithm config for quantization, can be 'DEFAULT', 'HQQ' or 'RTN', 'GPTQ' "
-                "(default DEFAULT when not set). For 4b quantize a model with RTN or GPTQ algorithm. "
-                "Please refer to https://github.com/intel/neural-compressor/blob/master/docs/source/quantization_weight_only.md "  # noqa: E501
+                description="'RTN' and 'GPTQ' are available from onnxruntime>=1.17.0 "
+                "'DEFAULT', 'HQQ' are available from onnxruntime>=1.18.0 (default DEFAULT when not set). "
```

Comment:
`default DEFAULT when not set` doesn't apply to our pass right? Cause if algorithm is None, we will skip weight only quant.